modules/supported-platforms-for-managed-clusters.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: PROCEDURE
+[id="supported-platforms-for-managed-clusters_{context}"]
+= supported-platforms-for-managed-clusters
+
+You can manually specify the platform type for managed clusters if you do not want the platform type to be set automatically.
+
+.Prerequisites
+
+The Assisted Installer currently supports the following OpenShift platforms:
+- BareMetal
+- VSphere
+- None
+
+.Procedure
+
+. Select the platform in the AgentClusterInstall CR, via `spec.platformType`.
+
```

Comment:
@jelkosz and @trewest, do you have an example code snippet that I can use for this step?

Reply:
```
apiVersion: extensions.hive.openshift.io/v1beta1
kind: AgentClusterInstall
metadata:
  name: cluster
  namespace: cluster
spec:
  platformType: VSphere
  
```

Reply:
Updated with requested code snippet.